### ISSUE TITLE
loss divergence

### ISSUE THREAD

**Issue by kwk2696** [s-sahoo/duo#11]
Thanks for the great work. 

I have a question regarding the loss function. The current objective minimizes the negative log-likelihood -log p(x), but this formulation seems prone to divergence. We've observed such loss blowing up during training. Would it be acceptable to replace the plain -log p(x) objective with PyTorch’s CrossEntropyLoss instead? Any guidance or alternative suggestions would be greatly appreciated.

**Comment by s-sahoo** [comment]
closing the issue for now
